bundles/org.openhab.core.semantics/model/SemanticTags.csv

```diff
@@ -35,18 +35,18 @@ Location,Garden,Outdoor,Garden,Gardens,
 Location,Patio,Outdoor,Patio,Patios,
 Location,Porch,Outdoor,Porch,Porches,
 Location,Terrace,Outdoor,Terrace,"Terraces, Deck, Decks",
-Point,Alarm,,Alarm,,
-Point,Calculation,,Calculation,,
-Point,Control,,Control,,
-Point,Switch,Control,Switch,,
-Point,Forecast,,Forecast,,
-Point,Measurement,,Measurement,,
-Point,Setpoint,,Setpoint,,
-Point,Status,,Status,,
+Point,Alarm,,Alarm,,Read only data point indicating an alarm condition
+Point,Calculation,,Calculation,,Read only analog calculated data point
+Point,Control,,Control,,Read-Write analog data point
+Point,Switch,Control,Switch,,Read-Write digital data point
+Point,Forecast,,Forecast,,Read only analog predicted data point
+Point,Measurement,,Measurement,,Read only analog data point from a sensor
+Point,Setpoint,,Setpoint,,Read-Write analog data point for a controller
```

Comment:
I don't know how to read this description, does it rule out read-only setpoints? In that case I think that would be a shame. IMHO a setpoint can be either writable or read-only. The latter for the cases where an API is not allowed to change it, but still can read it. One example is my Miele oven.

Reply:
The description does not rule out read only. But if you want different wording please say so.